golang/main.go

```
@@ -335,7 +348,10 @@ type WebhookRequest struct {
 }
 
 func webhook(w http.ResponseWriter, r *http.Request) {
-	b, _ := ioutil.ReadAll(r.Body)
+	b, err := io.ReadAll(r.Body)
+	if err != nil {
```

golangci-lint (1.64.8)

SA9003: empty branch

(staticcheck)